ui/tests/integration/components/form-field-test.js

```diff
@@ -121,6 +121,27 @@ module('Integration | Component | form field', function (hooks) {
     assert.ok(spy.calledWith('foo', expectedSeconds), 'onChange called with correct args');
   });
```

Comment:
in the original PR there is an import for dasherize on row 4 I don't see here?  Just a question, not blocking.